src/aws_encryption_sdk_cli/internal/arg_parsing.py

```diff
@@ -85,22 +86,33 @@ def add_argument(self, *args, **kwargs):
 
         return super(CommentIgnoringArgumentParser, self).add_argument(*args, **kwargs)
 
+    def __parse_line(self, arg_line):
+        # type: (ARGPARSE_TEXT) -> List[str]
+        """Parses a line of arguments into individual arguments intelligently for different platforms.
```

Comment:
Could you add a short note about what the intended behavior is? e.g. "Support single and double quotes for interpreting escape characters, and use backticks as escape on Windows"?